src/content/protocols/stakewise-v3/ethereum.md

```diff
@@ -0,0 +1,250 @@
+
+> Exit Window score: High
+
+## Accessibility
+
+StakeWise v3 can be accessed primarily through the official StakeWise user interface, which enables deposits, vault interactions, OsToken minting and redemption workflows. In addition to this interface, the protocol exposes all core functionality directly through on-chain contracts, allowing users or integrators to interact without relying on a single frontend.
+
+Crucially, the protocol provides an open-source SDK and a public vault-interface repository, which serves as a backup solution that allows users to locally host or deploy their own interface if the official website becomes unavailable. While the official interface remains the dominant access point, these tools—combined with the ability to submit transactions via block explorers guarantee that access to user positions is not dependent solely on the availability of the main domain.
+
+> Accessibility score: Medium
+
+## Conclusion
+
+StakeWise v3 receives High scores in the Upgradeability, Autonomy, and Exit Window dimensions. This classifies the protocol as Stage 0.
+
+The protocol could transition to Stage 1 by introducing a governance structure that enforces a mandatory exit window of at least seven days for all upgrades or parameter changes that affect user positions. To achieve this, the "instant upgrade" powers currently held by the Multisigs would need to be revoked or transferred to a time-locked contract, potentially reserving immediate intervention capabilities strictly for a dedicated Security Council handling provable emergencies.
+
+Progress toward Stage 2 would require minimizing reliance on privileged multisigs and external operators to the point where dependencies no longer hold direct influence over user funds. This would involve migrating key components to fully on-chain governance with 30-day delay periods, decentralizing the oracle and validator key management infrastructure, and removing any administrative pathways that allow operators to unilaterally alter system-critical logic.
+
+# Reviewer's Notes
+
+⚠️ During our analysis, we identified ...
+
+Some contracts (e.g., EthFoxVault, RewardEthToken) use ERC1967 or UUPS proxies. Upgrades are authorized by admin or \_authorizeUpgrade.
+
+The OsTokenFlashLoans contract was excluded from the core analysis because it operates as an auxiliary component and does not influence user custody or the protocol’s upgrade paths in a way that fits within the decentralization evaluation criteria.
+
+Because Stakewise is issuing tokens that may be used across external DeFi applications, user funds may become indirectly exposed to the risks of those external protocols. Partial withdrawals typically complete faster (≈27–28 hours), while full exits can take up to one month to finalize.
+
+# Protocol Analysis
+
+Here include the diagram. Please explain what the main contracts are doing within the diagram. graph TD User((User)) DAO[StakeWise DAO / Multisigs]
+
+# Dependencies
+
```

Comment:
This section is a bit AI-coded. As mentioned above only the oracle mechanism and validator infrastructure are considered external dependencies. What we want to know is who is operating those, and what impact can they have on the funds? You can have a look at Lido's dependencies section as a good example. This will requrie an analysis slightly more advanced than pure text